code/__DEFINES/inventory_sizes.dm

```diff
@@ -23,10 +23,10 @@
 
 //linear increase. Using many small storage containers is more space-efficient than using large ones,
 //in exchange for being limited in the w_class of items that will fit
-#define BASE_STORAGE_CAPACITY(w_class) (10*(w_class-1))
+#define BASE_STORAGE_CAPACITY(w_class) (5*(w_class-1))
```

Comment:
I would make this value 7, I'm not aware of any server which uses this low of a value.